laborator/content/etapele-compilarii/06-obj-link-dev/Makefile

```diff
@@ -1,2 +1,14 @@
-PROGNAME := main
-include ../../utils/Makefile.generic
+CC = gcc
+CFLAGS = -g -Wall -Wextra -Werror -fno-pic -masm=intel
+
+.PHONY: all clean
+
+all: main
+
+# ** TO DO **
+# Add dependecies here so that the Makefile works
```

Comment:
```suggestion
# TODO: Add dependecies here so that the Makefile works
```

Reply:
The commit you are referring it is not mine, I always used this workflow. I will change it for the next commits.